src/sysv/systemd/sssd.service.in

```diff
@@ -10,6 +10,9 @@ StartLimitBurst=5
 [Service]
 Environment=DEBUG_LOGGER=--logger=files
 EnvironmentFile=-@environment_file@
+ExecStartPre=+-/bin/chown -f @SSSD_USER@:@SSSD_USER@ /etc/sssd/sssd.conf
+ExecStartPre=+-/bin/chown -f -R @SSSD_USER@:@SSSD_USER@ /etc/sssd/conf.d
+ExecStartPre=+-/bin/chown -f -R @SSSD_USER@:@SSSD_USER@ /etc/sssd/pki
```

Comment:
Might be worth adding the /var/log/sssd as well just for a good measure.

Reply:
On the other hand, logs (and /var/lib/sss/db/* content) is different vs 'sssd.conf'

'sssd.conf' can be created by a 3rd party (realm, ansible script, etc)
logs and ldb-cache, on the other hand, are only created by SSSD itself.

I think it's reasonable to expect that in a rare case of re-configuration of SSSD service user, admin should wipe those artifacts as well. So doesn't justify additional "permanent" lines in service description, and I rather tend to remove corresponding lines from socket activated services.

Reply:
Ack. We just need to consider updates and have appropriate pieces in scriptlets to be able to upgrade/rollback existing configuration including  /var/lib/sss/db/ and logs (with the directory).

Reply:
https://github.com/SSSD/sssd/blob/ca684cd156debbdf3d95776271e7020b883b1e81/contrib/sssd.spec.in#L1111 ?